includes/components/class-admin.php

```diff
@@ -1135,12 +1136,36 @@ public function update_meta_box( $post_id ) {
 								}
 							}
 						} else {
-							wp_update_post(
-								[
-									'ID' => $post_id,
-									$field->get_arg( '_alias' ) => $field->get_value(),
-								]
-							);
+
+							// Set post arguments.
+							$args = [
```

Comment:
please check if there's another way to do this, admin component is used as an intermediary for the WP admin API and it shouldn't contain the custom email logic, if possible do this via hooks or another way